bosh/opsfiles/production.yml

```
@@ -123,3 +123,31 @@
 - type: replace
   path: /instance_groups/name=prometheus-tooling/vm_type
   value: m5.large
+
+# add some prod-only rules
+# monitor Federalist proxy
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: federalist-proxy
+    rules:
+    - alert: FederalistProxyUnhealthy
+      # get all the cf application instances with matching org, space, and app IDs. 
+      # Subtract from that all the instances with matching org, space, app ids with state Running.
+      # Take the minimum of that, and see if it's more than zero
+      expr: min(cf_application_instances{organization_id="((federalist_org_id))", space_id="((federalist_proxy_space_id))", application_id="((federalist_proxy_app_id))"} - cf_application_instances_running{state=~"STARTED", organization_id="((federalist_org_id))"", space_id="((federalist_proxy_space_id))"", application_id=((federalist_proxy_app_id))"} ) > 0
```

```suggestion
      expr: min(cf_application_instances{organization_id="((federalist_org_id))", space_id="((federalist_proxy_space_id))", application_id="((federalist_proxy_app_id))"} - cf_application_instances_running{state=~"STARTED", organization_id="((federalist_org_id))", space_id="((federalist_proxy_space_id))", application_id=((federalist_proxy_app_id))"} ) > 0
```

Also, I looked in the tooling credhub and I don't see those variables, can you make sure they exist before this gets merged?